javalib/src/main/scala/java/util/regex/PatternCompiler.scala

```diff
@@ -80,15 +82,15 @@ private[regex] object PatternCompiler {
 
   /** Cache for `Support.supportsUnicode`. */
   private val _supportsUnicode =
-    (esVersion >= ESVersion.ES2015) || featureTest("u")
+    (linkingInfo.esVersion >= ESVersion.ES2015) || featureTest("u")
```

Comment:
Any reason not to import this?

Reply:
Yes: `linkingInfo` is a `def`, so we cannot import members from it. I first tried an alias def, but I couldn't put it in `PatternCompiler` itself without destroying the constant folding (because `object PatternCompiler` is not pure), and it didn't feel right anywhere else.